src/protocol-reference.rst

```diff
@@ -713,6 +713,23 @@ Streaming instructions
     :arg string filename:
         The name of the file, as it would be saved on a filesystem.
 
+.. guac:instruction:: msg
+    :sent-by: server
+    :phase: interactive
+
+    Sends a message from the server (gaucd) to the client. The nature of these
+    messages is intentionally ambiguous and flexible - the message must include
+    a numeric code that the client understands and can act on, and may also
+    include one or more arguments that can be used by the client in association
+    with the message.
+
+    :arg integer msg:
+        A numeric value indicating the message that is being passed to the client.
+
+    :arg string args:
+        One or more arguments associated with the message that is being sent
+        to the client.
```

Comment:
Does the `msg` instruction also allows zero arguments? If so, I think this should be rephrased to "any number of arguments".

Reply:
Yeah, good point - fixed via rebase.